les/server_handler.go

```diff
@@ -865,9 +870,29 @@ func (h *serverHandler) handleMsg(p *peer, wg *sync.WaitGroup) error {
 				}
 			}()
 		}
+	case GetGatewayFeeMsg:
+		p.Log().Trace("Received gatewayFee request")
+		var req struct {
+			ReqID uint64
+		}
+		if err := msg.Decode(&req); err != nil {
+			return errResp(ErrDecode, "msg %v: %v", msg, err)
+		}
+
+		if h == nil || h.gatewayFee == nil {
```

Comment:
It is assumed that `h` is non-nil here, or if it is a very bad error has occured. You do not need to c
heck that `h` is `nil`. If `gatewayFee` is `nil`, a response should still be sent. The response should indicate tha
t ethe gateway fee is `0`